util-scripts/update-cabal-versions.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+if [ -z "$1" ]; then
+    echo "Supply single argument -- version to update CSL to"
+    exit
+fi
+
+newVersion=$1
+
+function updateVersion() {
+  sed -E -i -e "s/^(version\:\s+)(.+)/\1$newVersion/" $1 
+}
+
+updateVersion cardano-sl.cabal
+updateVersion core/cardano-sl-core.cabal
+updateVersion infra/cardano-sl-infra.cabal
+updateVersion db/cardano-sl-db.cabal
+updateVersion lrc/cardano-sl-lrc.cabal
```

Comment:
He-he, I thought you would do it be searching for all `.cabal` files so that we don't have to modify this script when we add new package. Anyway, we already have few scripts to modify, so it's not a big problem.